src/lib/CommandHandler.ts

```diff
@@ -124,14 +124,18 @@ export class CommandHandler
         if (!cmd)
             return undefined;
 
-        const launchInputs = vscode.workspace.getConfiguration().get('launch.inputs') || [];
-        const taskInputs = vscode.workspace.getConfiguration().get('tasks.inputs') || [];
-
         let inputs: any[] = [];
-        if (Array.isArray(launchInputs))
-            inputs = inputs.concat(launchInputs);
-        if (Array.isArray(taskInputs))
-            inputs = inputs.concat(taskInputs);
+        if (vscode.workspace.workspaceFolders) {
```

Comment:
Shouldn't it have a fallback for when the user is not using multi folder workspace?

Reply:
I first tried to leave the old code and just add my changes. But then all inputs were twice in the inputs array.

I tested it with a code-workspace with two folders AND when opening a single folder directly.